Algorithmia/algorithm.py

```diff
@@ -74,53 +66,36 @@ def publish(self, details={}, settings={}, version_info={}):
         versionRequestObj = VersionInfoPublish(**version_info)
         publish_parameters = {"details": detailsObj, "settings": settingsObj, "version_info": versionRequestObj}
         version_request = VersionRequest(**publish_parameters) # VersionRequest | Publish Version Request
-        try:
-            # Publish Algorithm
-            api_response = self.client.manageApi.publish_algorithm(self.username, self.algoname, version_request)
-            return api_response
-        except ApiException as e:
-            error_message = json.loads(e.body)
-            raise raiseAlgoApiError(error_message)
+        # Publish Algorithm
+        api_response = self.client.algorithmApi.publish_algorithm(self.username, self.algoname, version_request)
+        return api_response
 
     def builds(self, limit=56, marker=None):
-        try:
-            if marker is not None:
-                api_response = self.client.manageApi.get_algorithm_builds(self.username, self.algoname, limit=limit, marker=marker)
-            else:
-                api_response = self.client.manageApi.get_algorithm_builds(self.username, self.algoname, limit=limit)
-            return api_response
-        except ApiException as e:
-            error_message = json.loads(e.body)
-            raise raiseAlgoApiError(error_message)
+        if marker is not None:
+            api_response = self.client.buildsApi.get_algorithm_builds(self.username, self.algoname, limit=limit, marker=marker)
+        else:
+            api_response = self.client.buildsApi.get_algorithm_builds(self.username, self.algoname, limit=limit)
```

Comment:
Is this if-else necessary? Seems like same default argument?

Reply:
correct it can be removed!